src/pymovements/utils/checks.py

```diff
@@ -183,5 +193,10 @@ def check_is_length_matching(**kwargs) -> None:
     value_1 = kwargs[key_1]
     value_2 = kwargs[key_2]
 
+    # mypy does not understand value_1 and value_2 can't be None
```

Comment:
will this still be needed if `None` is not one of the accepted types (is not `Sized`)?

Reply:
no, but see comment above (https://github.com/aeye-lab/pymovements/pull/295#discussion_r1165202105)